### ISSUE TITLE
Yaru release for eoan

### ISSUE THREAD

**Issue by iainlane** [ubuntu/yaru#1505]
Hey there

Eoan's UI freeze is this Thursday - what do you think about tagging a new release before then to pick up all the latest changes?

**Comment by clobrano** [comment]
definitely, @Feichtmeier, @madsrh, @ubuntujaggers, is there any ready content to be merged?

**Comment by Feichtmeier** [comment]
Yes made two more PRs

- desktop icons rounding
- sound indicator fast mouse wheel move distortion

Would be nice to get those in, but please wait for @madsrh review on the sound theme

**Comment by Feichtmeier** [comment]
Also maybe this one
https://github.com/ubuntu/yaru/issues/1498

**Comment by clobrano** [comment]
I don't see the one for the soundindicator

**Comment by clobrano** [comment]
> Also maybe this one
> #1498

I think the only thing that can get in now is the upstream version

**Comment by Feichtmeier** [comment]
> I don't see the one for the soundindicator

Mads merged it already :dancer:
